burn_erc721Token.md

```diff
@@ -0,0 +1,74 @@
+# Burning TradeTrustERC721 Token
+
+## Status
+
+Draft
+
+## Goal
+
+ERC721 Burnable Token can be irreversibly burned (destroyed). The ERC721 implemented by OpenZeppelin allows this by transfering burn tokens to address(0) as shown below:
+
+```sol
+function _burn(address owner, uint256 tokenId) internal {
+    require(ownerOf(tokenId) == owner, "ERC721: burn of token that is not own");
+
+    _clearApproval(tokenId);
+
+    _ownedTokensCount[owner].decrement();
+    _tokenOwner[tokenId] = address(0);
+
+    emit Transfer(owner, address(0), tokenId);
+}
+```
+
+However, TradeTrust is still required to render and track burnt token. Since unminted tokens and destoryed token have the same owner address in the Erc721 Token Registry, we needed to find a way to differentiate a unminted and burnt token. This means that TradeTrust would have to modify the ERC721 implementation or modify its verifier method.
```

Comment:
```suggestion
In the TradeTrust application, tokens are still required to be traceable and viewable after destruction. Since unminted tokens and burnt token appear to have the same owner address (.ownerOf() throws error 'cannot query non-existent token') in the ERC721 Token Registry, it is necessary to find a way to differentiate an unminted and burnt token. 

There exists two broad approaches to resolve this:
1. The ERC721 implementation is modified to support the additional state of 'destroyed'
1. The verification method in OpenAttestation or TradeTrust is modified to detect the history of the token, if any exists
```